{
  "SchemaVersion": 2,
  "CreatedAt": "2025-10-01T21:48:59.326060279Z",
  "ArtifactName": "ghcr.io/example/phase0:latest",
  "ArtifactType": "container_image",
  "Metadata": {
    "OS": {
      "Family": "debian",
      "Name": "12.12"
    },
    "ImageID": "sha256:5bd69d849c130281d4cff85531fe696f489d6ba8a99181a856f10714347f2141",
    "DiffIDs": [
      "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e",
      "sha256:b3a19b5ba9637c600bb37a67b0a30d8349ebc8f7602fc9c46376f7855a9b3c57",
      "sha256:933d50692269a20734b5e6e54c70c009b27a46d8fdd90c8fbb8a7f3941c97e70",
      "sha256:ac1568d6117bb1cf7d91e2d4dc47b3b1907c568e1b60b6825599cf5763b1a0a8",
      "sha256:02492764898a76f9ca3bdd23b94dbc4ffbafff1fbcae27f11cf98c23f36e6bca",
      "sha256:51810e28921c0771fbb714f90c18f47f19af026f23318db4f8e61f9da54c0cc5",
      "sha256:77377fe9e8fbfcb227b5ed83b29a80d372977459a093234872a020a98e5d4740",
      "sha256:8b8c0e688d0bfd358853bd85ea4b6cd604ed5b457c9348819dc4042ce2a635db",
      "sha256:da114cc3e5e5e6bbc39096abbbc79e7b4e271349711beaf4d348bb934aaf4d87",
      "sha256:5f70bf18a086007016e948b04aed3b82103a36bea41755b6cddfaf10ace3c6ef",
      "sha256:83f610bb422a600517b65ee3e96c22d96e462fb5ba333aee2fbb978c5084b915",
      "sha256:2c907a77ddd72a03156aeba3d842a45c14ccd74f478e69010b1c9b805c5baede",
      "sha256:b749a2a9aed1547f0cca2476771b388ba99ace2c35b36b25db9ad93af3781641",
      "sha256:8133fdb4dfc0f3ffe44d249c86ba71d630e68de482f022752dcc9c219d1e7e38"
    ],
    "RepoTags": [
      "ghcr.io/example/phase0:latest"
    ],
    "ImageConfig": {
      "architecture": "amd64",
      "created": "2025-10-01T21:48:42.328719989Z",
      "history": [
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "# debian.sh --arch 'amd64' out/ 'bookworm' '@1759104000'",
          "comment": "debuerreotype 0.16"
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "ENV PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "ENV LANG=C.UTF-8",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "RUN /bin/sh -c set -eux; \tapt-get update; \tapt-get install -y --no-install-recommends \t\tca-certificates \t\tnetbase \t\ttzdata \t; \trm -rf /var/lib/apt/lists/* # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "ENV GPG_KEY=A035C8C19219BA821ECEA86B64E628F8D684696D",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "ENV PYTHON_VERSION=3.11.13",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "ENV PYTHON_SHA256=8fb5f9fbc7609fa822cb31549884575db7fd9657cbffb89510b5d7975963a83a",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "RUN /bin/sh -c set -eux; \t\tsavedAptMark=\"$(apt-mark showmanual)\"; \tapt-get update; \tapt-get install -y --no-install-recommends \t\tdpkg-dev \t\tgcc \t\tgnupg \t\tlibbluetooth-dev \t\tlibbz2-dev \t\tlibc6-dev \t\tlibdb-dev \t\tlibffi-dev \t\tlibgdbm-dev \t\tliblzma-dev \t\tlibncursesw5-dev \t\tlibreadline-dev \t\tlibsqlite3-dev \t\tlibssl-dev \t\tmake \t\ttk-dev \t\tuuid-dev \t\twget \t\txz-utils \t\tzlib1g-dev \t; \t\twget -O python.tar.xz \"https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz\"; \techo \"$PYTHON_SHA256 *python.tar.xz\" | sha256sum -c -; \twget -O python.tar.xz.asc \"https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc\"; \tGNUPGHOME=\"$(mktemp -d)\"; export GNUPGHOME; \tgpg --batch --keyserver hkps://keys.openpgp.org --recv-keys \"$GPG_KEY\"; \tgpg --batch --verify python.tar.xz.asc python.tar.xz; \tgpgconf --kill all; \trm -rf \"$GNUPGHOME\" python.tar.xz.asc; \tmkdir -p /usr/src/python; \ttar --extract --directory /usr/src/python --strip-components=1 --file python.tar.xz; \trm python.tar.xz; \t\tcd /usr/src/python; \tgnuArch=\"$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)\"; \t./configure \t\t--build=\"$gnuArch\" \t\t--enable-loadable-sqlite-extensions \t\t--enable-optimizations \t\t--enable-option-checking=fatal \t\t--enable-shared \t\t$(test \"${gnuArch%%-*}\" != 'riscv64' \u0026\u0026 echo '--with-lto') \t\t--with-ensurepip \t; \tnproc=\"$(nproc)\"; \tEXTRA_CFLAGS=\"$(dpkg-buildflags --get CFLAGS)\"; \tLDFLAGS=\"$(dpkg-buildflags --get LDFLAGS)\"; \tLDFLAGS=\"${LDFLAGS:--Wl},--strip-all\"; \tmake -j \"$nproc\" \t\t\"EXTRA_CFLAGS=${EXTRA_CFLAGS:-}\" \t\t\"LDFLAGS=${LDFLAGS:-}\" \t; \trm python; \tmake -j \"$nproc\" \t\t\"EXTRA_CFLAGS=${EXTRA_CFLAGS:-}\" \t\t\"LDFLAGS=${LDFLAGS:--Wl},-rpath='\\$\\$ORIGIN/../lib'\" \t\tpython \t; \tmake install; \t\tcd /; \trm -rf /usr/src/python; \t\tfind /usr/local -depth \t\t\\( \t\t\t\\( -type d -a \\( -name test -o -name tests -o -name idle_test \\) \\) \t\t\t-o \\( -type f -a \\( -name '*.pyc' -o -name '*.pyo' -o -name 'libpython*.a' \\) \\) \t\t\\) -exec rm -rf '{}' + \t; \t\tldconfig; \t\tapt-mark auto '.*' \u003e /dev/null; \tapt-mark manual $savedAptMark; \tfind /usr/local -type f -executable -not \\( -name '*tkinter*' \\) -exec ldd '{}' ';' \t\t| awk '/=\u003e/ { so = $(NF-1); if (index(so, \"/usr/local/\") == 1) { next }; gsub(\"^/(usr/)?\", \"\", so); printf \"*%s\\n\", so }' \t\t| sort -u \t\t| xargs -rt dpkg-query --search \t\t| awk 'sub(\":$\", \"\", $1) { print $1 }' \t\t| sort -u \t\t| xargs -r apt-mark manual \t; \tapt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \trm -rf /var/lib/apt/lists/*; \t\texport PYTHONDONTWRITEBYTECODE=1; \tpython3 --version; \t\tpip3 install \t\t--disable-pip-version-check \t\t--no-cache-dir \t\t--no-compile \t\t'setuptools==65.5.1' \t\t'wheel\u003c0.46' \t; \tpip3 --version # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "RUN /bin/sh -c set -eux; \tfor src in idle3 pip3 pydoc3 python3 python3-config; do \t\tdst=\"$(echo \"$src\" | tr -d 3)\"; \t\t[ -s \"/usr/local/bin/$src\" ]; \t\t[ ! -e \"/usr/local/bin/$dst\" ]; \t\tln -svT \"$src\" \"/usr/local/bin/$dst\"; \tdone # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-08-08T18:20:34Z",
          "created_by": "CMD [\"python3\"]",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:27Z",
          "created_by": "ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:27Z",
          "created_by": "WORKDIR /app",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:27Z",
          "created_by": "LABEL maintainer=Phase0 Foundation Team",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "COPY /install /usr/local # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "COPY src /app/src # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "COPY scripts /app/scripts # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "COPY docker/start.sh /app/scripts/start.sh # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "COPY scripts/healthcheck.py /app/scripts/healthcheck.py # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "RUN /bin/sh -c chmod +x /app/scripts/start.sh # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "RUN /bin/sh -c chmod +x /app/scripts/healthcheck.py # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "RUN /bin/sh -c groupadd --system appuser \u0026\u0026 useradd --system --gid appuser --create-home appuser # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "RUN /bin/sh -c chown -R appuser:appuser /app # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "USER appuser",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "ENV APP_ENV=development",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "ENV LOG_LEVEL=info",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "HEALTHCHECK \u0026{[\"CMD\" \"python\" \"/app/scripts/healthcheck.py\"] \"30s\" \"5s\" \"10s\" '\\x05'}",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-10-01T21:48:42Z",
          "created_by": "CMD [\"/app/scripts/start.sh\"]",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        }
      ],
      "os": "linux",
      "rootfs": {
        "type": "layers",
        "diff_ids": [
          "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e",
          "sha256:b3a19b5ba9637c600bb37a67b0a30d8349ebc8f7602fc9c46376f7855a9b3c57",
          "sha256:933d50692269a20734b5e6e54c70c009b27a46d8fdd90c8fbb8a7f3941c97e70",
          "sha256:ac1568d6117bb1cf7d91e2d4dc47b3b1907c568e1b60b6825599cf5763b1a0a8",
          "sha256:02492764898a76f9ca3bdd23b94dbc4ffbafff1fbcae27f11cf98c23f36e6bca",
          "sha256:51810e28921c0771fbb714f90c18f47f19af026f23318db4f8e61f9da54c0cc5",
          "sha256:77377fe9e8fbfcb227b5ed83b29a80d372977459a093234872a020a98e5d4740",
          "sha256:8b8c0e688d0bfd358853bd85ea4b6cd604ed5b457c9348819dc4042ce2a635db",
          "sha256:da114cc3e5e5e6bbc39096abbbc79e7b4e271349711beaf4d348bb934aaf4d87",
          "sha256:5f70bf18a086007016e948b04aed3b82103a36bea41755b6cddfaf10ace3c6ef",
          "sha256:83f610bb422a600517b65ee3e96c22d96e462fb5ba333aee2fbb978c5084b915",
          "sha256:2c907a77ddd72a03156aeba3d842a45c14ccd74f478e69010b1c9b805c5baede",
          "sha256:b749a2a9aed1547f0cca2476771b388ba99ace2c35b36b25db9ad93af3781641",
          "sha256:8133fdb4dfc0f3ffe44d249c86ba71d630e68de482f022752dcc9c219d1e7e38"
        ]
      },
      "config": {
        "Cmd": [
          "/app/scripts/start.sh"
        ],
        "Healthcheck": {
          "Test": [
            "CMD",
            "python",
            "/app/scripts/healthcheck.py"
          ],
          "Interval": 30000000000,
          "Timeout": 5000000000,
          "StartPeriod": 10000000000,
          "Retries": 5
        },
        "Env": [
          "PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
          "LANG=C.UTF-8",
          "GPG_KEY=A035C8C19219BA821ECEA86B64E628F8D684696D",
          "PYTHON_VERSION=3.11.13",
          "PYTHON_SHA256=8fb5f9fbc7609fa822cb31549884575db7fd9657cbffb89510b5d7975963a83a",
          "PYTHONDONTWRITEBYTECODE=1",
          "PYTHONUNBUFFERED=1",
          "APP_ENV=development",
          "LOG_LEVEL=info"
        ],
        "Labels": {
          "maintainer": "Phase0 Foundation Team"
        },
        "User": "appuser",
        "WorkingDir": "/app",
        "ArgsEscaped": true
      }
    }
  },
  "Results": [
    {
      "Target": "ghcr.io/example/phase0:latest (debian 12.12)",
      "Class": "os-pkgs",
      "Type": "debian",
      "Vulnerabilities": [
        {
          "VulnerabilityID": "CVE-2025-6020",
          "PkgID": "libpam-modules@1.5.2-6+deb12u1",
          "PkgName": "libpam-modules",
          "PkgIdentifier": {
            "PURL": "pkg:deb/debian/libpam-modules@1.5.2-6%2Bdeb12u1?arch=amd64\u0026distro=debian-12.12",
            "UID": "854c4c35b318f724"
          },
          "InstalledVersion": "1.5.2-6+deb12u1",
          "Status": "affected",
          "Layer": {
            "DiffID": "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-6020",
          "DataSource": {
            "ID": "debian",
            "Name": "Debian Security Tracker",
            "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
          },
          "Title": "linux-pam: Linux-pam directory Traversal",
          "Description": "A flaw was found in linux-pam. The module pam_namespace may use access user-controlled paths without proper protection, allowing local users to elevate their privileges to root via multiple symlink attacks and race conditions.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "oracle-oval": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/06/17/1",
            "https://access.redhat.com/errata/RHSA-2025:10024",
            "https://access.redhat.com/errata/RHSA-2025:10027",
            "https://access.redhat.com/errata/RHSA-2025:10180",
            "https://access.redhat.com/errata/RHSA-2025:10354",
            "https://access.redhat.com/errata/RHSA-2025:10357",
            "https://access.redhat.com/errata/RHSA-2025:10358",
            "https://access.redhat.com/errata/RHSA-2025:10359",
            "https://access.redhat.com/errata/RHSA-2025:10361",
            "https://access.redhat.com/errata/RHSA-2025:10362",
            "https://access.redhat.com/errata/RHSA-2025:10735",
            "https://access.redhat.com/errata/RHSA-2025:10823",
            "https://access.redhat.com/errata/RHSA-2025:11386",
            "https://access.redhat.com/errata/RHSA-2025:11487",
            "https://access.redhat.com/errata/RHSA-2025:14557",
            "https://access.redhat.com/errata/RHSA-2025:15099",
            "https://access.redhat.com/errata/RHSA-2025:15709",
            "https://access.redhat.com/errata/RHSA-2025:15827",
            "https://access.redhat.com/errata/RHSA-2025:15828",
            "https://access.redhat.com/errata/RHSA-2025:16524",
            "https://access.redhat.com/errata/RHSA-2025:9526",
            "https://access.redhat.com/security/cve/CVE-2025-6020",
            "https://bugzilla.redhat.com/2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2388220",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-6020",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-8941",
            "https://errata.almalinux.org/9/ALSA-2025-9526.html",
            "https://errata.rockylinux.org/RLSA-2025:14557",
            "https://linux.oracle.com/cve/CVE-2025-6020.html",
            "https://linux.oracle.com/errata/ELSA-2025-9526.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-6020",
            "https://ubuntu.com/security/notices/USN-7580-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-6020",
            "https://www.openwall.com/lists/oss-security/2025/06/17/1"
          ],
          "PublishedDate": "2025-06-17T13:15:21.66Z",
          "LastModifiedDate": "2025-09-23T20:15:33.53Z"
        },
        {
          "VulnerabilityID": "CVE-2025-6020",
          "PkgID": "libpam-modules-bin@1.5.2-6+deb12u1",
          "PkgName": "libpam-modules-bin",
          "PkgIdentifier": {
            "PURL": "pkg:deb/debian/libpam-modules-bin@1.5.2-6%2Bdeb12u1?arch=amd64\u0026distro=debian-12.12",
            "UID": "daab75fb9dca86e0"
          },
          "InstalledVersion": "1.5.2-6+deb12u1",
          "Status": "affected",
          "Layer": {
            "DiffID": "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-6020",
          "DataSource": {
            "ID": "debian",
            "Name": "Debian Security Tracker",
            "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
          },
          "Title": "linux-pam: Linux-pam directory Traversal",
          "Description": "A flaw was found in linux-pam. The module pam_namespace may use access user-controlled paths without proper protection, allowing local users to elevate their privileges to root via multiple symlink attacks and race conditions.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "oracle-oval": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/06/17/1",
            "https://access.redhat.com/errata/RHSA-2025:10024",
            "https://access.redhat.com/errata/RHSA-2025:10027",
            "https://access.redhat.com/errata/RHSA-2025:10180",
            "https://access.redhat.com/errata/RHSA-2025:10354",
            "https://access.redhat.com/errata/RHSA-2025:10357",
            "https://access.redhat.com/errata/RHSA-2025:10358",
            "https://access.redhat.com/errata/RHSA-2025:10359",
            "https://access.redhat.com/errata/RHSA-2025:10361",
            "https://access.redhat.com/errata/RHSA-2025:10362",
            "https://access.redhat.com/errata/RHSA-2025:10735",
            "https://access.redhat.com/errata/RHSA-2025:10823",
            "https://access.redhat.com/errata/RHSA-2025:11386",
            "https://access.redhat.com/errata/RHSA-2025:11487",
            "https://access.redhat.com/errata/RHSA-2025:14557",
            "https://access.redhat.com/errata/RHSA-2025:15099",
            "https://access.redhat.com/errata/RHSA-2025:15709",
            "https://access.redhat.com/errata/RHSA-2025:15827",
            "https://access.redhat.com/errata/RHSA-2025:15828",
            "https://access.redhat.com/errata/RHSA-2025:16524",
            "https://access.redhat.com/errata/RHSA-2025:9526",
            "https://access.redhat.com/security/cve/CVE-2025-6020",
            "https://bugzilla.redhat.com/2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2388220",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-6020",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-8941",
            "https://errata.almalinux.org/9/ALSA-2025-9526.html",
            "https://errata.rockylinux.org/RLSA-2025:14557",
            "https://linux.oracle.com/cve/CVE-2025-6020.html",
            "https://linux.oracle.com/errata/ELSA-2025-9526.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-6020",
            "https://ubuntu.com/security/notices/USN-7580-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-6020",
            "https://www.openwall.com/lists/oss-security/2025/06/17/1"
          ],
          "PublishedDate": "2025-06-17T13:15:21.66Z",
          "LastModifiedDate": "2025-09-23T20:15:33.53Z"
        },
        {
          "VulnerabilityID": "CVE-2025-6020",
          "PkgID": "libpam-runtime@1.5.2-6+deb12u1",
          "PkgName": "libpam-runtime",
          "PkgIdentifier": {
            "PURL": "pkg:deb/debian/libpam-runtime@1.5.2-6%2Bdeb12u1?arch=all\u0026distro=debian-12.12",
            "UID": "a4ba3bc9ed7e203b"
          },
          "InstalledVersion": "1.5.2-6+deb12u1",
          "Status": "affected",
          "Layer": {
            "DiffID": "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-6020",
          "DataSource": {
            "ID": "debian",
            "Name": "Debian Security Tracker",
            "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
          },
          "Title": "linux-pam: Linux-pam directory Traversal",
          "Description": "A flaw was found in linux-pam. The module pam_namespace may use access user-controlled paths without proper protection, allowing local users to elevate their privileges to root via multiple symlink attacks and race conditions.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "oracle-oval": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/06/17/1",
            "https://access.redhat.com/errata/RHSA-2025:10024",
            "https://access.redhat.com/errata/RHSA-2025:10027",
            "https://access.redhat.com/errata/RHSA-2025:10180",
            "https://access.redhat.com/errata/RHSA-2025:10354",
            "https://access.redhat.com/errata/RHSA-2025:10357",
            "https://access.redhat.com/errata/RHSA-2025:10358",
            "https://access.redhat.com/errata/RHSA-2025:10359",
            "https://access.redhat.com/errata/RHSA-2025:10361",
            "https://access.redhat.com/errata/RHSA-2025:10362",
            "https://access.redhat.com/errata/RHSA-2025:10735",
            "https://access.redhat.com/errata/RHSA-2025:10823",
            "https://access.redhat.com/errata/RHSA-2025:11386",
            "https://access.redhat.com/errata/RHSA-2025:11487",
            "https://access.redhat.com/errata/RHSA-2025:14557",
            "https://access.redhat.com/errata/RHSA-2025:15099",
            "https://access.redhat.com/errata/RHSA-2025:15709",
            "https://access.redhat.com/errata/RHSA-2025:15827",
            "https://access.redhat.com/errata/RHSA-2025:15828",
            "https://access.redhat.com/errata/RHSA-2025:16524",
            "https://access.redhat.com/errata/RHSA-2025:9526",
            "https://access.redhat.com/security/cve/CVE-2025-6020",
            "https://bugzilla.redhat.com/2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2388220",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-6020",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-8941",
            "https://errata.almalinux.org/9/ALSA-2025-9526.html",
            "https://errata.rockylinux.org/RLSA-2025:14557",
            "https://linux.oracle.com/cve/CVE-2025-6020.html",
            "https://linux.oracle.com/errata/ELSA-2025-9526.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-6020",
            "https://ubuntu.com/security/notices/USN-7580-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-6020",
            "https://www.openwall.com/lists/oss-security/2025/06/17/1"
          ],
          "PublishedDate": "2025-06-17T13:15:21.66Z",
          "LastModifiedDate": "2025-09-23T20:15:33.53Z"
        },
        {
          "VulnerabilityID": "CVE-2025-6020",
          "PkgID": "libpam0g@1.5.2-6+deb12u1",
          "PkgName": "libpam0g",
          "PkgIdentifier": {
            "PURL": "pkg:deb/debian/libpam0g@1.5.2-6%2Bdeb12u1?arch=amd64\u0026distro=debian-12.12",
            "UID": "75408269dcf74277"
          },
          "InstalledVersion": "1.5.2-6+deb12u1",
          "Status": "affected",
          "Layer": {
            "DiffID": "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-6020",
          "DataSource": {
            "ID": "debian",
            "Name": "Debian Security Tracker",
            "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
          },
          "Title": "linux-pam: Linux-pam directory Traversal",
          "Description": "A flaw was found in linux-pam. The module pam_namespace may use access user-controlled paths without proper protection, allowing local users to elevate their privileges to root via multiple symlink attacks and race conditions.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "oracle-oval": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/06/17/1",
            "https://access.redhat.com/errata/RHSA-2025:10024",
            "https://access.redhat.com/errata/RHSA-2025:10027",
            "https://access.redhat.com/errata/RHSA-2025:10180",
            "https://access.redhat.com/errata/RHSA-2025:10354",
            "https://access.redhat.com/errata/RHSA-2025:10357",
            "https://access.redhat.com/errata/RHSA-2025:10358",
            "https://access.redhat.com/errata/RHSA-2025:10359",
            "https://access.redhat.com/errata/RHSA-2025:10361",
            "https://access.redhat.com/errata/RHSA-2025:10362",
            "https://access.redhat.com/errata/RHSA-2025:10735",
            "https://access.redhat.com/errata/RHSA-2025:10823",
            "https://access.redhat.com/errata/RHSA-2025:11386",
            "https://access.redhat.com/errata/RHSA-2025:11487",
            "https://access.redhat.com/errata/RHSA-2025:14557",
            "https://access.redhat.com/errata/RHSA-2025:15099",
            "https://access.redhat.com/errata/RHSA-2025:15709",
            "https://access.redhat.com/errata/RHSA-2025:15827",
            "https://access.redhat.com/errata/RHSA-2025:15828",
            "https://access.redhat.com/errata/RHSA-2025:16524",
            "https://access.redhat.com/errata/RHSA-2025:9526",
            "https://access.redhat.com/security/cve/CVE-2025-6020",
            "https://bugzilla.redhat.com/2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2372512",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2388220",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-6020",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-8941",
            "https://errata.almalinux.org/9/ALSA-2025-9526.html",
            "https://errata.rockylinux.org/RLSA-2025:14557",
            "https://linux.oracle.com/cve/CVE-2025-6020.html",
            "https://linux.oracle.com/errata/ELSA-2025-9526.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-6020",
            "https://ubuntu.com/security/notices/USN-7580-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-6020",
            "https://www.openwall.com/lists/oss-security/2025/06/17/1"
          ],
          "PublishedDate": "2025-06-17T13:15:21.66Z",
          "LastModifiedDate": "2025-09-23T20:15:33.53Z"
        },
        {
          "VulnerabilityID": "CVE-2025-7458",
          "PkgID": "libsqlite3-0@3.40.1-2+deb12u2",
          "PkgName": "libsqlite3-0",
          "PkgIdentifier": {
            "PURL": "pkg:deb/debian/libsqlite3-0@3.40.1-2%2Bdeb12u2?arch=amd64\u0026distro=debian-12.12",
            "UID": "84263b51d2906e6b"
          },
          "InstalledVersion": "3.40.1-2+deb12u2",
          "Status": "affected",
          "Layer": {
            "DiffID": "sha256:933d50692269a20734b5e6e54c70c009b27a46d8fdd90c8fbb8a7f3941c97e70"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-7458",
          "DataSource": {
            "ID": "debian",
            "Name": "Debian Security Tracker",
            "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
          },
          "Title": "sqlite: SQLite integer overflow",
          "Description": "An integer overflow in the sqlite3KeyInfoFromExprList function in SQLite versions 3.39.2 through 3.41.1 allows an attacker with the ability to execute arbitrary SQL statements to cause a denial of service or disclose sensitive information from process memory via a crafted SELECT statement with a large number of expressions in the ORDER BY clause.",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "cbl-mariner": 2,
            "nvd": 4,
            "redhat": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
              "V3Score": 9.1
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:L",
              "V3Score": 6.1
            }
          },
          "References": [
            "https://access.redhat.com/security/cve/CVE-2025-7458",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-7458",
            "https://sqlite.org/forum/forumpost/16ce2bb7a639e29b",
            "https://sqlite.org/src/info/12ad822d9b827777",
            "https://www.cve.org/CVERecord?id=CVE-2025-7458"
          ],
          "PublishedDate": "2025-07-29T13:15:28.953Z",
          "LastModifiedDate": "2025-08-11T19:11:30.693Z"
        },
        {
          "VulnerabilityID": "CVE-2023-45853",
          "PkgID": "zlib1g@1:1.2.13.dfsg-1",
          "PkgName": "zlib1g",
          "PkgIdentifier": {
            "PURL": "pkg:deb/debian/zlib1g@1.2.13.dfsg-1?arch=amd64\u0026distro=debian-12.12\u0026epoch=1",
            "UID": "cfe2e20187a599f5"
          },
          "InstalledVersion": "1:1.2.13.dfsg-1",
          "Status": "will_not_fix",
          "Layer": {
            "DiffID": "sha256:aca83606673032726b42f8e1396ceb979c32bfb26b602732baf699053e46b33e"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2023-45853",
          "DataSource": {
            "ID": "debian",
            "Name": "Debian Security Tracker",
            "URL": "https://salsa.debian.org/security-tracker-team/security-tracker"
          },
          "Title": "zlib: integer overflow and resultant heap-based buffer overflow in zipOpenNewFileInZip4_6",
          "Description": "MiniZip in zlib through 1.3 has an integer overflow and resultant heap-based buffer overflow in zipOpenNewFileInZip4_64 via a long filename, comment, or extra field. NOTE: MiniZip is not a supported part of the zlib product. NOTE: pyminizip through 0.2.6 is also vulnerable because it bundles an affected zlib version, and exposes the applicable MiniZip code through its compress API.",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 4,
            "ghsa": 4,
            "nvd": 4,
            "photon": 4,
            "redhat": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:L/I:L/A:L",
              "V3Score": 5.3
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2023/10/20/9",
            "http://www.openwall.com/lists/oss-security/2024/01/24/10",
            "https://access.redhat.com/security/cve/CVE-2023-45853",
            "https://chromium.googlesource.com/chromium/src/+/d709fb23806858847131027da95ef4c548813356",
            "https://chromium.googlesource.com/chromium/src/+/de29dd6c7151d3cd37cb4cf0036800ddfb1d8b61",
            "https://github.com/madler/zlib/blob/ac8f12c97d1afd9bafa9c710f827d40a407d3266/contrib/README.contrib#L1-L4",
            "https://github.com/madler/zlib/commit/73331a6a0481067628f065ffe87bb1d8f787d10c",
            "https://github.com/madler/zlib/pull/843",
            "https://github.com/smihica/pyminizip",
            "https://github.com/smihica/pyminizip/blob/master/zlib-1.2.11/contrib/minizip/zip.c",
            "https://lists.debian.org/debian-lts-announce/2023/11/msg00026.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2023-45853",
            "https://pypi.org/project/pyminizip/#history",
            "https://security.gentoo.org/glsa/202401-18",
            "https://security.netapp.com/advisory/ntap-20231130-0009",
            "https://security.netapp.com/advisory/ntap-20231130-0009/",
            "https://ubuntu.com/security/notices/USN-7107-1",
            "https://www.cve.org/CVERecord?id=CVE-2023-45853",
            "https://www.winimage.com/zLibDll/minizip.html"
          ],
          "PublishedDate": "2023-10-14T02:15:09.323Z",
          "LastModifiedDate": "2024-12-20T17:41:31.237Z"
        }
      ]
    },
    {
      "Target": "Python",
      "Class": "lang-pkgs",
      "Type": "python-pkg",
      "Vulnerabilities": [
        {
          "VulnerabilityID": "CVE-2024-6345",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.11/site-packages/setuptools-65.5.1.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@65.5.1",
            "UID": "7c14745fc6f5094d"
          },
          "InstalledVersion": "65.5.1",
          "FixedVersion": "70.0.0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:933d50692269a20734b5e6e54c70c009b27a46d8fdd90c8fbb8a7f3941c97e70"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6345",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools",
          "Description": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-94"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6726",
            "https://access.redhat.com/security/cve/CVE-2024-6345",
            "https://bugzilla.redhat.com/2297771",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2297771",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6345",
            "https://errata.almalinux.org/9/ALSA-2024-6726.html",
            "https://errata.rockylinux.org/RLSA-2024:5530",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/commit/88807c7062788254f654ea8c03427adc859321f0",
            "https://github.com/pypa/setuptools/pull/4332",
            "https://huntr.com/bounties/d6362117-ad57-4e83-951f-b8141c6e7ca5",
            "https://linux.oracle.com/cve/CVE-2024-6345.html",
            "https://linux.oracle.com/errata/ELSA-2024-6726.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6345",
            "https://ubuntu.com/security/notices/USN-7002-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6345"
          ],
          "PublishedDate": "2024-07-15T01:15:01.73Z",
          "LastModifiedDate": "2024-11-21T09:49:28.17Z"
        },
        {
          "VulnerabilityID": "CVE-2025-47273",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.11/site-packages/setuptools-65.5.1.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@65.5.1",
            "UID": "7c14745fc6f5094d"
          },
          "InstalledVersion": "65.5.1",
          "FixedVersion": "78.1.1",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:933d50692269a20734b5e6e54c70c009b27a46d8fdd90c8fbb8a7f3941c97e70"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-47273",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "setuptools: Path Traversal Vulnerability in setuptools PackageIndex",
          "Description": "setuptools is a package that allows users to download, build, install, upgrade, and uninstall Python packages. A path traversal vulnerability in `PackageIndex` is present in setuptools prior to version 78.1.1. An attacker would be allowed to write files to arbitrary locations on the filesystem with the permissions of the process running the Python code, which could escalate to remote code execution depending on the context. Version 78.1.1 fixes the issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:L",
              "V3Score": 7.1
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:13578",
            "https://access.redhat.com/security/cve/CVE-2025-47273",
            "https://bugzilla.redhat.com/2366982",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2366982",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-47273",
            "https://errata.almalinux.org/9/ALSA-2025-13578.html",
            "https://errata.rockylinux.org/RLSA-2025:11036",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2025-49.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/6ead555c5fb29bc57fe6105b1bffc163f56fd558/setuptools/package_index.py#L810C1-L825C88",
            "https://github.com/pypa/setuptools/commit/250a6d17978f9f6ac3ac887091f2d32886fbbb0b",
            "https://github.com/pypa/setuptools/issues/4946",
            "https://github.com/pypa/setuptools/security/advisories/GHSA-5rjg-fvgr-3xxf",
            "https://linux.oracle.com/cve/CVE-2025-47273.html",
            "https://linux.oracle.com/errata/ELSA-2025-9940.html",
            "https://lists.debian.org/debian-lts-announce/2025/05/msg00035.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-47273",
            "https://ubuntu.com/security/notices/USN-7544-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-47273"
          ],
          "PublishedDate": "2025-05-17T16:15:19.11Z",
          "LastModifiedDate": "2025-06-12T16:29:01.66Z"
        }
      ]
    }
  ]
}
